fast-enrichment-service/tests/utils/singapore-assertions.ts

```
@@ -0,0 +1,88 @@
+import { expect } from '@playwright/test';
+import { GenericAssertions } from '@gpp/pw-core';
+import { SINGAPORE_BANK_CODES, SINGAPORE_TEST_CONFIG } from '../fixtures/singapore/test-accounts';
+
+export class SingaporeAssertions extends GenericAssertions {
+  
+  /**
+   * Validates that XML contains Singapore-specific elements
+   */
+  static expectValidSingaporeXML(xmlContent: string, expect: any): void {
+    this.expectValidXML(xmlContent, expect);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)